### PR TITLE
Jureid/walkthrough clean

### PR DESCRIPTION
**Pull request by jureid**
Removed the Build your C++ file task in the walkthrough and updated instructions for Run and debug your C++ file. Instead of telling customers to create tasks.json and launch.json themselves, we tell them to just try pressing F5. Then we describe what the newly created tasks.json and launch.json files do.

### CODE REVIEW

**Review by michelleangela — APPROVED** [review]

File: Extension/walkthrough/debugconfig/run-and-debug-project-linux.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+1. Open the C++ source file that you want to run and debug. Make sure this file is active (currently displayed and selected) in the editor.
 
-3. Choose **g++ build and debug active file.**
+2.  Press `F5`. Or, from the main menu, choose **Run > Start Debugging**.
 
-    ![Dropdown showing C++ debug configurations for GCC on Linux](linux-build-and-debug-active-file.png)
+3. Select **C++ (GDB/LLDB)**.
 
-4. To start a debugging session, press `F5` or from the main menu choose **Run > Start Debugging**.
+After running and debugging your C++ file for the first time, you'll notice two new files inside your project's **.vscode** folder: **tasks.json** and **launch.json**.
```
Comment (sean-mcmanus) [inline Extension/walkthrough/debugconfig/run-and-debug-project-linux.md:13]: Isn't there another dropdown/step to choose the compiler to use?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

File: Extension/package.json [hunk 2]
```diff
@@ -112,40 +112,10 @@
...
             }
           },
-          {
-            "id": "build.project.mac",
-            "title": "Build your C++ file",
-            "description": "Create a default build task for your project. \n[Learn more](https://code.visualstudio.com/docs/cpp/config-clang-mac#_build-helloworldcpp)",
-            "when": "isMac",
-            "media": {
-              "altText": "A markdown file with instructions for configuring the default build task on macOS with Clang.",
-              "path": "walkthrough/buildtask/build-project-mac.md"
```
Comment (Colengms) [inline Extension/package.json:122]: Should these MD files be deleted?

**Review by Colengms — CHANGES_REQUESTED** [review]

File: Extension/walkthrough/debugconfig/run-and-debug-project-linux.md (hunk 1 above)
Comment (jureid) [reply Extension/walkthrough/debugconfig/run-and-debug-project-linux.md:13]: You're right! I'll add that in.

Comment (jureid) [reply Extension/walkthrough/debugconfig/run-and-debug-project-linux.md:13]: You're right! I'll add that in.

File: Extension/package.json (hunk 2 above)
Comment (jureid) [reply Extension/package.json:122]: Yes, will do that

**Review by sean-mcmanus — APPROVED** [review]

**Review by Colengms — APPROVED** [review]

**Pull request merged**